sys/include/color.h

```diff
@@ -104,6 +105,30 @@ void color_str2rgb(const char *str, color_rgb_t *color);
  */
 void color_rgb2str(const color_rgb_t *rgb, char *str);
 
+/**
+ * @brief Invert a given rgb color
+ *
+ * @param[in] rgb       Input rgb color, that should be converted. Must be NOT NULL
```

Comment:
Could you please use the `@pre` tag for that (either additionally or alternatively; your choice ;-))